includes/OAuth/OAuthAuthenticator.php

```diff
@@ -10,7 +11,8 @@
 
 namespace GbWeiss\includes\OAuth;
 
-defined('ABSPATH') || exit;
```

Comment:
We have too keep this line to prevent direct access. It shuld be right below the namespace.